### PR TITLE
user interface pages [#160420163] [#160422356] [#160419653]

### PR DESCRIPTION
**Pull request by kaggwachristopher**
User interface pages for fast food fast

### CODE REVIEW

**Comment by KicaRonaldOkello** [comment]
Please use atomic commits. I believe your pivotal tracker can have morethan 2 stories for this challenge

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+	background: -webkit-linear-gradient(top,red,orange);
+     border: 0px;
+     font-family: vivaldi;
+     font-size: 23px;
+     color: white;
+	width: 420px;
+	border-radius: 10px;
+	padding: 10px;
+}
+html, body{
```
Comment (jeanjoe) [inline css/login.css]: Am seeing lots of this lines of code from line #1  to #99 repeated.

**Comment by Luleherll** [comment]
well organized

**Comment by KicaRonaldOkello** [comment]
images are not stored in the css folder. i'd rather you created an img folder separate

**Pull request merged**